R/knit_print.R

```diff
@@ -7,13 +7,37 @@
 #'
 #' @keywords internal
 #' @noRd
-knit_print.fontawesome <- function(x, ...) {
+knit_print.fontawesome <- function(x, ..., options, inline = FALSE) {
 
   # nocov start
 
   if (requireNamespace("knitr", quietly = TRUE)) {
```

Comment:
I'd prefer there to be an early return, since the `else` condition is so far away.

```R
if (!requireNamespace("knitr", quietly = TRUE)) {
  stop(....)
}
```

Reply:
Do we want to error on that ? I just followed @rich previous idea to just not print anything. 

But for all that matter, I beleive when `knit_print` function will be called it means **knitr** is already installed and in used. So basically, I would have just not test it if the function was not already like that before this PR - We could also remove this completely. or wrong idea and not clean enough ?

Reply:
Oh right, that makes sense. I agree we can just remove that check.